pkg/scheduler/schedulercache/node_info.go

```diff
@@ -542,11 +543,36 @@ func (n *NodeInfo) updateImageSizes() {
 	for _, image := range node.Status.Images {
 		for _, name := range image.Names {
 			imageSizes[name] = image.SizeBytes
+			for _, alias := range getImageNameAliases(name) {
+				imageSizes[alias] = image.SizeBytes
```

Comment:
`imageSizes` use string as key, but seems alias is a `[]string`

Reply:
>imageSizes use string as key, but seems alias is a []string

alias is a string :)

Reply:
> Actually, you don't have to do a monkey patch, we already have a pkg/util/parsers/parsers.go :-)

I agree! I was concerned with (as always :-) a full-fledged image name parsing could add performance overhead, unnecessarily; so the "monkey patch" essentially does a stripped-down" version of the parsing.

I applied the patch as you suggested - it's cleaner, thank you @resouer  👍